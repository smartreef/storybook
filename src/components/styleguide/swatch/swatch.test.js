import React from 'react';
import renderer from 'react-test-renderer';
import ColourSwatch from './';

it('match snapshot', () => {
	const component = renderer.create(<ColourSwatch />);
	const tree = component.toJSON();

	expect(tree).toMatchSnapshot();
});
