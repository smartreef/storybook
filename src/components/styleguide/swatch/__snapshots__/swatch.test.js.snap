// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div
  className="flex items-center mt-5"
  onClick={[Function]}
>
  <div
    class="h-12 w-12 rounded-lg shadow-inner bg-black"
  />
  <div
    className="ml-2 text-gray-800 text-xs leading-none pl-1"
  >
    <div
      className="font-semibold"
    >
      Black
    </div>
    <div
      className="mt-1 font-normal opacity-75"
    >
      #000000
    </div>
  </div>
</div>
`;
